### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by cpanato**
- Update readme to add content on how to use the GitHub Action

Need this PR first: https://github.com/sigstore/cosign-installer/pull/3

/assign @dlorenc

### CODE REVIEW

**Comment by dlorenc** [comment]
This action is going to be soo good! Thanks @cpanato!

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
-# cosign-installer
\ No newline at end of file
+# cosign-installer GitHub Action
+
+This action enables you to sign, verify container images. For a quick start of `cosign`, please refer to https://github.com/sigstore/cosign#quick-start
+
+## Usage
+
+Add the following entry to your Github workflow YAML file with the required inputs:
```
Comment (mbestavros) [inline README.md:7]: This sentence says that the `cosign-release` input is required, but it isn't _strictly_ required -- and further down the README, we claim that it's optional as well. Clarifying the wording here would be helpful.

**Review by mbestavros — CHANGES_REQUESTED** [review]
One minor nitpick, but nothing major -- this looks good.

A suggestion: it may be helpful to link to one of our test workflows to show an example of what an actual working workflow looks like. Not strictly necessary since we provide an example in this README, though.

File: README.md (hunk 1 above)
Comment (cpanato) [reply README.md]: done, thanks for your review and feedback

**Comment by cpanato** [comment]
@dlorenc @mbestavros if y'all agree after this I think we can tag a release 1.0.0 or 0.1.0 and publish that to the marketplace.

to publish that some admin of the org/repo will need to accept some things, but that is straight to do, and when creating a new release will have a checkbox to publish that to the marketplace

**Comment by dlorenc** [comment]
> to publish that some admin of the org/repo will need to accept some things, but that is straight to do, and when creating a new release will have a checkbox to publish that to the marketplace

Yup sounds good!

**Pull request merged**